src/__test__/persistentSubscription/deletePersistentSubscriptionToAll.test.ts

```diff
@@ -0,0 +1,73 @@
+/** @jest-environment ./src/__test__/utils/enableVersionCheck.ts */
+
+import { createTestNode, matchServerVersion, optionalDescribe } from "../utils";
+
+import {
+  EventStoreDBClient,
+  persistentSubscriptionToAllSettingsFromDefaults,
+  UnsupportedError,
+} from "../..";
+
+describe("deletePersistentSubscriptionToAll", () => {
+  const supported = matchServerVersion`>=21.10`;
+  const node = createTestNode();
+  let client!: EventStoreDBClient;
+
+  beforeAll(async () => {
+    await node.up();
+
+    client = new EventStoreDBClient(
+      {
+        endpoint: node.uri,
+      },
+      { rootCertificate: node.rootCertificate },
+      { username: "admin", password: "changeit" }
+    );
+  });
+
+  afterAll(async () => {
+    await node.down();
+  });
+
+  optionalDescribe(!supported)("Not Supported (<21.10)", () => {
```

Comment:
**CONCERN:** Don't we have persistent subscriptions to `$all` supported from 21.6? See the release notes: https://www.eventstore.com/blog/21.6.0-release-notes. Do we have a reason to limit it to above 21.10?

Reply:
As discussed, 21.6 doesn't have support for the server features api, so persistent subscriptions to all with 21.6 are not supported by this client. This dissension will be communicated in release notes.